### ISSUE TITLE
TypeError: (0 , entities_1.decodeHTML) is not a function

### ISSUE THREAD

**Issue by BrentonCozby** [aws/aws-sdk-js-v3#3975]
### Checkboxes for prior research

- [X] I've gone through [Developer Guide](https://docs.aws.amazon.com/sdk-for-javascript/v3/developer-guide) and [API reference](https://docs.aws.amazon.com/AWSJavaScriptSDK/v3/latest)
- [X] I've checked [AWS Forums](https://forums.aws.amazon.com) and [StackOverflow](https://stackoverflow.com/questions/tagged/aws-sdk-js).
- [X] I've searched for [previous similar issues](https://github.com/aws/aws-sdk-js-v3/issues) and didn't find any solution.

### Describe the bug

I'm using `DeleteObjectsCommand` from `'@aws-sdk/client-s3'`. It works to delete all the objects I give it, but it throws the following error afterwards, every time:

```console
TypeError: (0 , entities_1.decodeHTML) is not a function
    at Object.tagValueProcessor (/usr/src/server/node_modules/@aws-sdk/client-s3/dist-cjs/protocols/Aws_restXml.js:10072:115)
    at processTagValue (/usr/src/server/node_modules/fast-xml-parser/src/xmlstr2xmlnode.js:78:19)
    at Object.getTraversalObj (/usr/src/server/node_modules/fast-xml-parser/src/xmlstr2xmlnode.js:199:31)
    at exports.parse (/usr/src/server/node_modules/fast-xml-parser/src/parser.js:19:39)
    at /usr/src/server/node_modules/@aws-sdk/client-s3/dist-cjs/protocols/Aws_restXml.js:10067:55
    at processTicksAndRejections (node:internal/process/task_queues:96:5)
    at async deserializeAws_restXmlDeleteObjectsCommand (/usr/src/server/node_modules/@aws-sdk/client-s3/dist-cjs/protocols/Aws_restXml.js:3588:87)
    at async /usr/src/server/node_modules/@aws-sdk/middleware-serde/dist-cjs/deserializerMiddleware.js:7:24
    at async /usr/src/server/node_modules/@aws-sdk/middleware-signing/dist-cjs/middleware.js:11:20
    at async StandardRetryStrategy.retry (/usr/src/server/node_modules/@aws-sdk/middleware-retry/dist-cjs/StandardRetryStrategy.js:51:46) {
  '$metadata': { attempts: 1, totalRetryDelay: 0 }
}
```

Here's the `DeleteObjectsCommandInput` value:
```json
{
  "Bucket": "my-bucket-name",
  "Delete": {
    "Objects": [
      {
        "Key": "photos/some-content-hash-xs.webp"
      },
      {
        "Key": "photos/some-content-hash-sm.webp"
      },
      {
        "Key": "photos/some-content-hash-md.webp"
      }
    ]
  }
}
```

### SDK version number

@aws-sdk/client-s3@3.171.0

### Which JavaScript Runtime is this issue in?

Node.js

### Details of the browser/Node.js/ReactNative version

v16.14.2

### Reproduction Steps

```ts
// typescript v4.7.2

import {
  S3Client,
  DeleteObjectsCommand,
} from '@aws-sdk/client-s3'

const myAccessKeyId = 'myAccessKeyId'
const mySecretAccessKey = 'mySecretAccessKey'
const myRegion = 'myRegion'
const myBucket = 'myBucket'

let client: S3Client

export function getClient() {
  if (!client) {
    client = new S3Client({
      credentials: {
        accessKeyId: myAccessKeyId,
        secretAccessKey: mySecretAccessKey,
      },
      region: myRegion,
    })
  }

  return client
}

export function deleteObjects(args: {
  keys: string[]
}) {
  const { keys } = args

  const command = new DeleteObjectsCommand({
    Bucket: myBucket,
    Delete: {
      Objects: keys.map((key) => ({ Key: key })),
    },
  })

  return getClient().send(command)
}

// in another file:
deleteObjects({ keys: ['key1', 'key2', 'key3'] }) // this successfully deletes the objects, but also throws said error
```

### Observed Behavior

Objects are successfully deleted and an error is thrown:

```console
TypeError: (0 , entities_1.decodeHTML) is not a function
    at Object.tagValueProcessor (/usr/src/server/node_modules/@aws-sdk/client-s3/dist-cjs/protocols/Aws_restXml.js:10072:115)
    at processTagValue (/usr/src/server/node_modules/fast-xml-parser/src/xmlstr2xmlnode.js:78:19)
    at Object.getTraversalObj (/usr/src/server/node_modules/fast-xml-parser/src/xmlstr2xmlnode.js:199:31)
    at exports.parse (/usr/src/server/node_modules/fast-xml-parser/src/parser.js:19:39)
    at /usr/src/server/node_modules/@aws-sdk/client-s3/dist-cjs/protocols/Aws_restXml.js:10067:55
    at processTicksAndRejections (node:internal/process/task_queues:96:5)
    at async deserializeAws_restXmlDeleteObjectsCommand (/usr/src/server/node_modules/@aws-sdk/client-s3/dist-cjs/protocols/Aws_restXml.js:3588:87)
    at async /usr/src/server/node_modules/@aws-sdk/middleware-serde/dist-cjs/deserializerMiddleware.js:7:24
    at async /usr/src/server/node_modules/@aws-sdk/middleware-signing/dist-cjs/middleware.js:11:20
    at async StandardRetryStrategy.retry (/usr/src/server/node_modules/@aws-sdk/middleware-retry/dist-cjs/StandardRetryStrategy.js:51:46) {
  '$metadata': { attempts: 1, totalRetryDelay: 0 }
}
```

### Expected Behavior

Objects are successfully deleted and no errors are thrown

**Comment by yenfryherrerafeliz** [comment]
Hi @BrentonCozby, thanks for opening this issue. Could you please let me know if you have tried with the latest version of the SDK?. I just tried to reproduce the reported behavior but I got not issues. Below are the details of my reproduction:
- SDK version: 3.178.0
- Code:
```js
import {S3Client, DeleteObjectsCommand} from "@aws-sdk/client-s3";

const client = new S3Client({
    region: 'REGION'
});
const keys = ["key1.txt", "key2.txt", "key3.txt"];
const response = await client.send(new DeleteObjectsCommand({
    Bucket: 'BUCKET',
    Delete: {
        Objects: keys.map((key) => ({Key: key}))
    }
}));

console.log('Response for deleting objects: ', response);

```
- Response I got:
```console
Response for deleting objects:  {
  '$metadata': {
    httpStatusCode: 200,
    requestId: undefined,
    extendedRequestId: 'HIDDEN',
    cfId: undefined,
    attempts: 1,
    totalRetryDelay: 0
  },
  Deleted: [
    {
      Key: 'key2.txt',
      VersionId: undefined,
      DeleteMarker: undefined,
      DeleteMarkerVersionId: undefined
    },
    {
      Key: 'key1.txt',
      VersionId: undefined,
      DeleteMarker: undefined,
      DeleteMarkerVersionId: undefined
    },
    {
      Key: 'key3.txt',
      VersionId: undefined,
      DeleteMarker: undefined,
      DeleteMarkerVersionId: undefined
    }
  ]
}

```
Thanks!

**Comment by BrentonCozby** [comment]
Bump

**Comment by BrentonCozby** [comment]
@yenfryherrerafeliz I tested with v3.183.0 and I'm no longer seeing the error. Thanks. Closing...

**Comment by BrentonCozby** [comment]
@yenfryherrerafeliz I'm still seeing the same error (same reproduction steps) using v3.183.0

<img width="1212" alt="image" src="https://user-images.githubusercontent.com/5513633/194131117-6fd0e4f7-42de-4c06-b5af-26106237ef4f.png">

**Comment by kuhe** [comment]
`entities.decodeHTML` is a function exported by the package `entities`@`2.2.0`, which the `@aws-sdk/client-s3` package declares as a direct dependency. 

Could you try the following please?
- confirm that entities v2.2.0 is installed in the same node_modules context as `@aws-sdk/client-s3`.
  - you could try `npm ls entities`
- use a breakpoint or logging to discover what the value of `entities_1` is in your JS code.

**Comment by BrentonCozby** [comment]
@kuhe 

<img width="497" alt="image" src="https://user-images.githubusercontent.com/5513633/198684627-4d40a26d-3b77-4bc7-82e2-ed317b00cfc6.png">

**Comment by BrentonCozby** [comment]
@kuhe I'm closing this because the error has fixed itself after upgrading to v3.198.0 and reinstalling node_modules

<img width="396" alt="image" src="https://user-images.githubusercontent.com/5513633/198735074-23d64c27-3e93-4237-a546-00374c29635f.png">
